R/rwebppl.R

```diff
@@ -272,7 +272,6 @@ run_webppl <- function(program_code = NULL, program_file = NULL, data = NULL,
 
   # find location of rwebppl JS script, within rwebppl R package
   script_path <- file.path(rwebppl_path(), "js/rwebppl")
-  add_packages <- packages
```

Comment:
the only reason for this was a lifelong habit of not messing with the values of function inputs